README.md

```diff
@@ -20,6 +22,25 @@ The project is organized in the following modules:
 - `lint` - contains custom Lint checks
 - `sample` - a sample app for testing the Gradle plugin
 
+## Table of contents
+
+* [Requirements](#requirements)
+* [Usage](#usage)
+* [Getting started](#getting-started)
+    - [Options](#options)
+    - [Lint checks](#lint-checks)
+    - [Debug UI](#debug-ui)
+    - [Tasks](#tasks)
+* [Contributing](#contributing)
+* [License](#license)
+* [Credits](#credits)
+
+## Requirements
+
+This plugin has been written in Kotlin but works both inside Kotlin and Java projects.
+Minimum required API level to use Collar is 19.
+
+
```

Comment:
@andronicus-kim please recheck this. 
Also, is KAPT or something like this requierment to use collar?